### ISSUE TITLE
Keyboard navigation between staves broken on Multi-Bar rests

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#33789]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

If the user uses keyboard navigation to move on to a multi-bar rest they can then not use Alt + Up or Down arrows to move to other staves.  This is especially confusing if the score begins with a multi-bar rest and the user then doesn't realise there is more than 1 staff present.

STEPS:
1.  Open the attached score (or create one with at least 2 instruments and beginning with a multi-bar rest).
2.  Press Right then left arrow to hear the multi-bar rest read.
3. Now Press Alt + Down arrow to attempt to move down onto the second staff (oboe on the attached score).  The screen reader says "Down note in chord" and the user will assume there is only 1 staff.

[multi-bar rests.zip](https://github.com/user-attachments/files/28861247/multi-bar.rests.zip)

### Supporting files, videos and screenshots

As above

### In which versions of MuseScore Studio is this issue present?

4.7.2

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
